### DISCUSSION TITLE
Possibly silly question about series

### DISCUSSION THREAD

**Discussion by Kyther** [nianeyna/ao3downloader#95] [Q&A] [answered]
I'm wanting to make sure I understand the behavior of choosing 'n' at the "get works from series links" option.  I can't tell if "Note that this will cause the program to ignore *all* series links" means it will ignore even the ones on the page link I gave the script or not.  Like, if I give it a link to a fandom's page, and on that page is a fic that is part of a series--it will download that fic?  Or will it go "oh, that fic is part of a series" and skip it?  I figure it's the former but wanted to get that clarified, lol.

**Comment by nianeyna** [comment, marked as answer]
Ha, yes, it will still download fics that happen to be part of a series *as long as* the link that is on the page goes to the work directly. Basically that's just a warning for people who are downloading their bookmarks pages specifically, and have bookmarked a series (since you can do that) instead of the individual work/s in the series. If you're downloading a tag or a search result, "n" is usually the right choice for that option.
